mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -50,9 +50,9 @@ private[shared] object SharedParamsCodeGen {
         isValid = "ParamValidators.inRange(0, 1)", finalMethods = false),
       ParamDesc[Array[Double]]("thresholds", "Thresholds in multi-class classification" +
         " to adjust the probability of predicting each class." +
-        " Array must have length equal to the number of classes, with values >= 0." +
+        " Array must have length equal to the number of classes, with values > 0." +
         " The class with largest value p/t is predicted, where p is the original probability" +
-        " of that class and t is the class' threshold",
+        " of that class and t is the class's threshold",
         isValid = "(t: Array[Double]) => t.forall(_ >= 0)", finalMethods = false),
```

Comment:
This line doesn't line up with what is in `sharedParams.scala`. This file _should_ generate `sharedParams.scala` via `build/sbt "mllib/runMain org.apache.spark.ml.param.shared.SharedParamsCodeGen"`. `t.forall(_ >= 0)` Should be `t.forall(_ > 0)`.